app/src/main/java/com/a1ishka/bankaccount/domain/AccountDao.kt

```diff
@@ -0,0 +1,23 @@
+package com.a1ishka.bankaccount.domain
```

Comment:
dao is a part of Room. So it must be in data package

Reply:
moved to data (I thought it belong to domaun because repository (interfaces) has dependency of DAO and data depends on domain but no domain on data)